troposphere/ec2.py

```diff
@@ -916,3 +916,73 @@ class LaunchTemplate(AWSObject):
         'LaunchTemplateData': (LaunchTemplateData, False),
         'LaunchTemplateName': (basestring, False),
     }
+
+
+class FleetLaunchTemplateSpecificationRequest(AWSProperty):
+    props = {
+        'LaunchTemplateId': (basestring, False),
+        'LaunchTemplateName': (basestring, False),
+        'Version': (basestring, False),
+    }
+
+
+class FleetLaunchTemplateOverridesRequest(AWSProperty):
+    props = {
+        'AvailabilityZone': (basestring, False),
+        'InstanceType': (basestring, False),
+        'MaxPrice': (basestring, False),
+        'Priority': (double, False),
+        'SubnetId': (basestring, False),
+        'WeightedCapacity': (double, False),
+    }
+
+
+class FleetLaunchTemplateConfigRequest(AWSProperty):
+    props = {
+        'LaunchTemplateSpecification': (
+            FleetLaunchTemplateSpecificationRequest,
+            False
+        ),
+        'Overrides': (FleetLaunchTemplateOverridesRequest, False),
```

Comment:
The docs say this should be a list: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-ec2fleet-fleetlaunchtemplateconfigrequest.html#cfn-ec2-ec2fleet-fleetlaunchtemplateconfigrequest-overrides

Reply:
you're right, thanks for your review.